maintainers/maintainer-list.nix

```diff
@@ -1,21 +1,41 @@
 /* List of NixOS maintainers.
 
     handle = {
-      name = "Real name";
+      # Required
+      name = "Your name";
       email = "address@example.org";
+
+      # Optional
       github = "GithubUsername";
+      keys = [{
+        longkeyid = "rsa2048/0x0123456789ABCDEF";
+        fingerprint = "AAAA BBBB CCCC DDDD EEEE  FFFF 0000 1111 2222 3333";
```

Comment:
I know it's getting complicated but how about adding an expiry date for the key? That way if there are multiple listed it makes it clearer which one is only there for historical reason (and that no new stuff should be signed with it).